shared/ui-components/src/atoms/ApplicationStatus.tsx

```diff
@@ -1,9 +1,9 @@
 import * as React from "react"
 import moment from "moment"
-import t from "@bloom-housing/ui-components/src/helpers/translator"
+import { t } from "@bloom-housing/ui-components/src/helpers/translator"
```

Comment:
This should be a relative import, since we're inside the `ui-components` package:

```suggestion
import { t } from "../helpers/translator"
```